languages/cs.ini

```
@@ -427,6 +427,7 @@ Exception = "Výjimka"
 Excerpt = "Výňatek"
 exclude_facet = "[vynechat]"
 exclude_newspapers = "Bez novinových článků"
+excluded = "Vyloučeno:"
```

```suggestion
excluded = "Bez:"
```
Updating translation per request from @xmorave2